rechunker/api.py

```diff
@@ -81,7 +174,28 @@ def rechunk(
             )
             stores_delayed.append(delayed)
 
-        return stores_delayed
+        # This next block makes a task that
```

Comment:
Hopefully this makes sense. LMK if I can clarify anything.

Reply:
It's a bit above my pay grade! Just trying to learn what is going on here...